### PR TITLE
rpi5-image-xt: Make sure boot files deployed before do_image

### PR DESCRIPTION
**Pull request by dsemenets**
Make sure boot files deployed before do_image. This should minimize or eliminate issue when boot firmware files don't deploy.

### CODE REVIEW

File: recipes-core/image/rpi5-image-minimal-domd.bb [hunk 1]
```diff
@@ -7,6 +7,13 @@ EXTRA_IMAGE_FEATURES += "package-management"
```
Comment (oleksiimoisieiev) [inline recipes-core/image/rpi5-image-minimal-domd.bb:7]: commit message top looks long. Have you run checkpatch?

**Review by lorc — COMMENTED** [review]
Could you please provide more details? 

What was the root cause? Is this a proper fix or just a workaround?

**Comment by dsemenets** [comment]
I don't know the root cause. Perhaps this is Yocto issue. This patch performs do_deploy operation for rpi-bootfiles and stuff needed for rpi-bootfiles before do_image. Looks like this is proper fix. I tested it and didn't notice this issue but before I met it many times.

**Comment by lorc** [comment]
> Looks like this is proper fix.

How it can proper fix if 

> I don't know the root cause.

?

It *may* be a proper fix. But we can't know for sure, because we don't know what exactly caused the issue. We even can't call it "workaround" because we don't know what we are working around. So, this is a hack. 

It also looks like a hack, because you are changing image recipe to fix issues in u-boot recipes. 

I am asking you to replace "[experiment]" tag with "[HACk]"

**Comment by dsemenets** [comment]
Code like this used in raspberrypi meta-layer in sdcard image creation. We don't use raspberrypi's wic image so this code doesn't work for us.

**Comment by lorc** [comment]
Well, this certainly should be mentioned in the commit message.

**Comment by dsemenets** [comment]
> Well, this certainly should be mentioned in the commit message.
Done

**Review by lorc — APPROVED** [review]
Thanks.

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Pull request merged**